custom/opm/migrations/0001_drop_old_fluff_tables.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+from sqlalchemy import Table, MetaData
+from corehq.db import connection_manager
+from django.db import migrations
+
+
+def drop_tables(apps, schema_editor):
+    # show SQL commands
+    logging.getLogger('sqlalchemy.engine').setLevel(logging.INFO)
+
+    metadata = MetaData(bind=connection_manager.get_engine())
+
+    for table_name in [
+        'fluff_OPMHierarchyFluff',
+        'fluff_OpmCaseFluff',
+        'fluff_OpmFormFluff',
+        'fluff_OpmHealthStatusAllInfoFluff',
+        'fluff_VhndAvailabilityFluff',
+    ]:
+        Table(table_name, metadata).drop()
```

Comment:
don't think it's better to fail hard (since anyone who installs HQ after this migration was added should never have those tables). Also, this is why the build is failing.